tests/Expressions/operators.twig

```diff
@@ -28,6 +28,12 @@
 {{ a ?? b }}
 {{ a <=> b }}
 
+Some variation of spread operator
+{{ ...some_var }}
+<twig:Component {{ ...vars }} />
```

Comment:
I don't think this is a valid usage of spread operator. Haven't tried on real project and can't find it in twig documentation.

Reply:
I agree, I don't think this is valid usage of the operator.

Reply:
Nope, this is a valid syntax used by twig component.

Ref: https://symfony.com/bundles/ux-twig-component/current/index.html#component-html-syntax